docs/source/includes/gcloud_setup.rst

```diff
@@ -1 +1,24 @@
 Follow the Windows, Mac OS X or Linux instructions to install gcloud on your local machine: https://cloud.google.com/sdk/
+
```

Comment:
Thanks Ilia. I totally agree with this all in principle. It is often a poor user experience to have instructions that send you off to other instructions. But worse is when the instructions you have don't work. 

As a note - while calling "gcloud auth login" here will certainly work, the cloud/sdk instructions now direct you to call:

  gcloud init

This changed at the end of last month.

I have some more thoughts on this, that I'd like to discuss offline. I think we can take a little time to set up some process so that we can have the user experience we want and avoid getting out of date.